src/behaviors/Visibility/Visibility.js

```diff
@@ -171,108 +171,76 @@ export default class Visibility extends Component {
   }
 
   calculations = {
-    topPassed: false,
     bottomPassed: false,
-    topVisible: false,
     bottomVisible: false,
     fits: false,
     passing: false,
-    onScreen: false,
     offScreen: false,
+    onScreen: false,
+    topPassed: false,
+    topVisible: false,
   }
 
-  firedCallbacks = []
+  occurredCallbacks = []
```

Comment:
I think the original name here is more appropriate, `firedCallbacks`.

Reply:
Restored :+1: